mosaic-site/src/layouts/LayoutColumns/LayoutColumns.module.css

```diff
@@ -0,0 +1,39 @@
+.fullWidth {
+  max-width: 100%;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.withSidebar {
+  width: 100%;
+  z-index: 1;
+  display: flex;
+}
+
+.mainWrapper {
+  width: 100%;
+  min-width: 0;
```

Comment:
is this min-w needed?

Reply:
I believe this is required at least for now because of the way the sidebar has been styled. I can't seem to attach a screenshot but I'll ping you